### PR TITLE
Lo 11 create crud for products

### PR DESCRIPTION
**Pull request by Diana9218**
Create Crud for products

### CODE REVIEW

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                ->withSuccess('El nuevo producto fue agregado satisfactoriamente.');
+    }
+
+    /**
+     * Display the specified resource.
+     */
+    public function show(Product $product): View
+    {
+        return view('products.show', [
+            'product' => $product
```
Comment (zahirolamadrid) [inline app/Http/Controllers/ProductController.php]: Here you can use compact like in the line 59

File: app/Providers/AppServiceProvider.php [hunk 2]
```diff
@@ -19,6 +19,6 @@ public function register(): void
      */
     public function boot(): void
     {
-        //
+        //  Paginator::useBootstrapFive();
```
Comment (zahirolamadrid) [inline app/Providers/AppServiceProvider.php]: Remove the comments

File: app/Http/Controllers/ProductController.php (hunk 1 above)
Comment (Diana9218) [reply app/Http/Controllers/ProductController.php]: Done

File: app/Providers/AppServiceProvider.php (hunk 2 above)
Comment (Diana9218) [reply app/Providers/AppServiceProvider.php]: Done

File: resources/views/products/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                        @csrf
+                        @method('DELETE')
+                        <button type="submit" onclick="return confirm('Do you want to delete this product?');">Borrar</button>
+                    </form>
+                </td>
+            </tr>
+            @empty
+                <td colspan="6">
+                                <span class="text-danger">
+                                    <strong>No Product Found!</strong>
```
Comment (zahirolamadrid) [inline resources/views/products/index.blade.php]: This should be in Spanish

File: resources/views/products/show.blade.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<x-layout>
+    <div>
+        <h1><label>{{$product->name}}</label></h1>
+    </div>
+    <div>
+        <h3><label>Descripción del producto:</label></h3>
```
Comment (zahirolamadrid) [inline resources/views/products/show.blade.php]: This header is unnecessary

Comment (zahirolamadrid) [reply resources/views/products/show.blade.php]: image

Comment (Diana9218) [reply resources/views/products/show.blade.php]: Done

File: resources/views/products/index.blade.php (hunk 3 above)
Comment (Diana9218) [reply resources/views/products/index.blade.php]: Done

**Review by Luismgi — APPROVED** [review]

File: resources/views/products/show.blade.php (hunk 4 above)
Comment (Diana9218) [reply resources/views/products/show.blade.php]: Captura de pantalla (1)

Comment (Diana9218) [reply resources/views/products/show.blade.php]: Captura de pantalla (18)

File: resources/views/products/show.blade.php [hunk 5]
```diff
@@ -3,8 +3,7 @@
         <h1><label>{{$product->name}}</label></h1>
     </div>
     <div>
-        <h3><label>Descripción del producto:</label></h3>
```
Comment (yoyi1073) [inline resources/views/products/show.blade.php]: When you use the elements h1 and h3 you dont have to use label, they are already labels

Comment (Diana9218) [reply resources/views/products/show.blade.php]: Done

**Review by zahirolamadrid — APPROVED** [review]

**Review by yoyi1073 — APPROVED** [review]

**Pull request merged**